kitsune/wiki/tasks.py

```diff
@@ -214,6 +214,7 @@ def rebuild_kb():
 
 
 @shared_task(rate_limit="5/m")
+@skip_if_read_only_mode
```

Comment:
this is called by `rebuild_kb` task which already has the decorator. Maybe that's not the cause of the sentry error?

Reply:
Thanks - sorry, I had pulled this back into In Progress status for this very reason. Revision added.